server/metrics.go

```diff
@@ -134,7 +121,7 @@ func Timed(handler http.Handler, name string, p provider.Provider) *Timer {
 func (t *Timer) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	if !t.isProm {
 		defer func(start time.Time) {
-			t.Observe(time.Since(start))
+			t.Observe(float64(time.Since(start).Nanoseconds()) / 1000000.0)
```

Comment:
Ah! I had a brain fart last night and kept thinking `Seconds()` would round to the nearest second.

Thanks for reviewing this, @basvanbeek! I'm updating the PR now.